src/components/landing/Information.jsx

```diff
@@ -0,0 +1,25 @@
+import Image from "next/image";
+import image from "/public/transparent_logo_1.webp";
+
+const Information = () => {
+  return (
+    <div className="flex items-center">
+      <div className="w-1/2 ml-4">
+        <Image
+          className="object-contain w-full h-auto"
+          src={image}
+          alt="Logo 1"
+        />
+      </div>
+      <div className="w-7/12 px-4">
+        <p className="text-4xl leading-tight text-center my-8">
+          AI at UCR is the hub for all things Artificial Intelligence. We are
+          committed to introducing AI and ML enthusiasts to the wider horizons
+          of this evolving field.
+        </p>
+      </div>
+    </div>
```

Comment:
you made sure this will compile? usually you can not return two separate div tags like this
this is a genuine question, please respond either here or on discord

Reply:
It did compile on my end when i did npm run dev

I also tried to add it to the homepage to see if it would appear without looking too funky and I can see the logo with the text formatted the same way as the picture. 

Let me know if I'm supposed to check it any other way to make sure it compiles!

Reply:
![Screenshot (349)](https://github.com/acm-ucr/aiatucr-website/assets/83476722/7fc0319e-2a7f-4fab-aca8-39f58a497347)